.changeset/perfect-cobras-bake.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-jenkins-backend': minor
+'@backstage/plugin-jenkins': minor
+'example-app': minor
```

Comment:
Small change: No changes to Example App yet added to changeset. Please remove this line

Reply:
done @NishkarshRaj